### ISSUE TITLE
Inline demos that don't use JavaScript, still need an empty <script></script>

### ISSUE THREAD

**Issue by roelvangils** [ThePacielloGroup/infusion#10]
I wondered why my inline demo didn't work:

```
{{<demo>}}
  <section>
    <h1 id="myHeading">Great stuff</h1>
    <p>Just some text</p>
    <a href="http://www.example.com/" aria-describedby="greatStuff">More…</a>
  </section>
{{</demo>}}
```

Then I figured out I needed to add an empty <script></script>:

```
{{<demo>}}
  <section>
    <h1 id="myHeading">Great stuff</h1>
    <p>Just some text</p>
    <a href="http://www.example.com/" aria-describedby="greatStuff">More…</a>
  </section>
  <script></script>
{{</demo>}}
```

Perhaps you can make it so that this is not possible, since not all inline demos will require JavaScript to work.

**Comment by Heydon** [comment]
@roelvangils Yep, that was an oversight. Will take a look at fixing ASAP.

**Comment by Heydon** [comment]
@roelvangils Fixed with b7d0f50
